scripts/generate-report-pdf.py

```
@@ -88,7 +88,18 @@
 
 MARKER = "---REPORT-MARKER---"
 
-POSTS_SQL = """
+def _env_bool(name: str, default: bool = False) -> bool:
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)